libcodechecker/generic_package_suppress_handler.py

```diff
@@ -82,4 +82,5 @@ def get_suppressed(self, bug):
 
         return any([suppress for suppress in self.__suppress_info
                     if suppress[0] == bug['hash_value'] and
-                    suppress[1] == os.path.basename(bug['file_path'])])
+                    suppress[1] == os.path.basename(bug['file_path']) and
+                    suppress[3] in ['false_positive', 'intentional']])
```

Comment:
The same question here about `codechecker_suppress`?

Reply:
The status can be one of the following values:
- `false_positive`
- `intentional`
- `confirmed`

Note: `codechecker_suppress` source code comment will be marked as `false_positive`.

https://github.com/Ericsson/codechecker/blob/master/libcodechecker/source_code_comment_handler.py